test/e2e/tests/test_process/test_container.py

```diff
@@ -4,6 +4,7 @@
 from sinspqa.docker import get_container_id
 
 sinsp_args = [
+    "-j",
```

Comment:
@Molter73 after all do we still need to set it here or do I recall correctly that it's now set already before? Seems to not hurt specifying it here.

Reply:
It's auto-added here so it's not needed to be set in the tests:
https://github.com/falcosecurity/libs/blob/27579290443cf2589ad4fe872b8eaf4469605f9e/test/e2e/tests/commons/sinspqa/sinsp.py#L350-L352

@gnosek up to you if you want to remove or keep the line.

Reply:
Ty @Molter73 ACK if we have time we can do a quick cleanup afterwards.